### DISCUSSION TITLE
NewPipe association Plenum + board meeting: 16.12!

### DISCUSSION THREAD

**Discussion by Poolitzer** [TeamNewPipe/NewPipe#10591] [Announcements]
## [Meeting takes place here](https://meet.assassinate-you.net/NewPipeEVMeeting)

Hello fellow NewPipers,

We proudly announce our first association plenum meeting, and the second board meeting! We will start on the 16th of December at 16:00 (15:00 UTC), expect to be done with the board meeting at 18:00 (17:00 UTC); take half an hour break and start with the plenum meeting at 18:30 (17:30 UTC), all German time.

Only members of the association are allowed to vote in the plenum, and only board members in the board meeting.

The topics are:

**NewPipe e.V. board meeting Agenda**

- prelude
  - elect session leader (one of the board members)
  - elect person who records decisions of the board for the protocol (one of the board members)
  - acknowledging that the proper invite happened a week prior
  - last minute topics (Hint: No NewPipe related topics, please only NewPipe e.V. topics)
  - acknowledge topics
- topics
  - accept protocol from last meeting
  - money flow
  - IT situation
  - last minute:
    - from poolitzer: Synonym support
  - miscellaneous

- session end

**NewPipe e.V. Plenum Agenda**

- prelude
  - elect session leader (board member or one voted by the assembly)
  - elect person who records the protocol (one of the board members)
  - acknowledging that the proper invite happened two weeks prior
  - last minute topics (Hint: No NewPipe related topics, please only NewPipe e.V. topics)
  - acknowledge topics
- topics
  - accept protocol from last meeting
  - yearly report
  - banking report
  - discharge of the board
  - election of the board + cash auditor
  - logo
  - Psoidonym support
  - offer Newpipe website repo etc.
  - about employing people to work on NewPipe
  - miscellaneous
- session end

We will publish the link here before the meeting starts, I hope to see lots of people there.

Till then,

Poolitzer.

**Comment by Poolitzer** [comment]
Plenum addition topic, as suggested by the assassin: Do we want to offer to host NewPipes website and website repo + CI to automate deplyment

**Comment by Poolitzer** [comment]
Board meeting topic, by me, initiated by assassin: Add official pseudonym support, self written form (to accommodate this change), and if yes we also need to append our data protection guideline (which includes informing our member in the next plenum assembly)

**Comment by roland-5** [comment]
Was it recorded to hear/see it?

**Reply by Poolitzer** [reply]
No, thankfully. You also wouldn't want to hear me ramble on for minutes without stopping ;P

**Reply by Stypox** [reply]
We will publish the meetings protocol as soon as we have a proper website

**Comment by noahUKUK** [comment]
In a person's thinking, the answer can represent clarity, direction, or resolution. It may symbolize a breakthrough in understanding or a reaffirmation of beliefs. Ultimately[,](https://youtubvanced.com/) it can lead to deeper introspection, guiding future decisions and fostering a sense of purpose and empowerment.
